monitoring/uss_qualifier/rid/test_executor.py

```diff
@@ -1,17 +1,17 @@
 import json
 import uuid
-from typing import List, Optional
+from typing import List
 from monitoring.uss_qualifier.rid.aircraft_state_replayer import TestHarness, TestBuilder
-from monitoring.uss_qualifier.rid.utils import RIDQualifierTestConfiguration, InjectedFlight
+from monitoring.uss_qualifier.rid.utils import RIDQualifierTestConfiguration, InjectedFlight, FullFlightRecord
 from monitoring.uss_qualifier.rid import display_data_evaluator, reports
 from monitoring.monitorlib.infrastructure import DSSTestSession
 from monitoring.monitorlib.auth import make_auth_adapter
 
 
-def main(
-        test_configuration: RIDQualifierTestConfiguration,
-        auth_spec: str, aircraft_state_files: Optional[list] = None) -> reports.Report:
-    my_test_builder = TestBuilder(test_configuration=test_configuration, aircraft_state_files=aircraft_state_files)
+def test_rid(test_configuration: RIDQualifierTestConfiguration,
```

Comment:
Do you think it would possible to find a better name to limit the usage of test_* functions and file names? It can be misleading for new comers since it overlaps with [pytest naming conventions](https://docs.pytest.org/en/latest/explanation/goodpractices.html#conventions-for-python-test-discovery).

Reply:
Yes -- done.

Reply:
Yes -- done.